dataprofiler/tests/data_readers/test_data_utils.py

```diff
@@ -37,7 +37,17 @@ def test_file_UTF_encoding_detection(self):
             detected_encoding = \
                 data_utils.detect_file_encoding(file_path=input_file["path"])
             self.assertEqual(detected_encoding.lower(), input_file["encoding"])
-
+        
+        try:
+            from charset_normalizer import CharsetNormalizerMatches as CnM
+            detected_encoding = \
+                data_utils.detect_file_encoding(file_path=os.path.join(test_dir, 'txt/utf8.txt'))
+            self.assertEqual("utf-8", detected_ecnoding.lower()) 
+        except:
+            detected_encoding = \
+                data_utils.detect_file_encoding(file_path=os.path.join(test_dir, 'txt/utf8.txt'))
+            self.assertEqual("windows-1254", detected_encoding.lower()) 
```

Comment:
@lettergram I added a test that uses the utf8.txt file from https://github.com/stain/encoding-test-files. Currently, I have it set to use CharsetNormalizer if it can and assert that it's detected as utf-8. Otherwise, it should be detected as a windows-1254 file by chardet.

Reply:
I would leave a comment on this case to explain that one of the detections is incorrect, but it is knowingly incorrect in order to show that the CharsetNormalizer is superior and should be tried first.

Reply:
Added a comment to clarify that we are looking to verify that chardet returns an incorrect detection as compared to charset_normalizer